Dockerfile

```diff
@@ -1,93 +1,72 @@
-# Dockerfile to build an image able to compile flashlight for a variety of
-# platforms.
-#
-# > make docker
+# This docker machine is able to compile and sign Lantern for Linux and
+# Windows.
 
 FROM fedora:21
-MAINTAINER "José Carlos Nieto" <xiam@getlantern.org>
+MAINTAINER "The Lantern Team" <team@getlantern.org>
 
-ENV GO_VERSION go1.4.3
-ENV GOROOT_BOOTSTRAP /go1.4
-ENV GOROOT /go
-ENV GOPATH /
-
-ENV PATH $PATH:$GOROOT/bin
 ENV WORKDIR /lantern
+ENV SECRETS /secrets
 
-# Go binary for bootstrapping.
-ENV GO_PACKAGE_URL https://storage.googleapis.com/golang/go1.4.3.linux-amd64.tar.gz
+RUN mkdir -p $WORKDIR
+RUN mkdir -p $SECRETS
 
 # Updating system.
-RUN yum -y update && yum clean all
+RUN yum update -y && yum makecache && yum clean packages
+RUN yum install -y deltarpm && yum clean packages
```

Comment:
I though that `deltarpm` was like a compressed bindiff, but I guess you're right, it seems like in this particular case it will only work for `yum update`. I've just added that line, thanks!

The `-C` flag would case yum to look into the cached data only but that's not what we want. `makecache` should reduce the times yum tries to update its database, it should read packages and dependencies from disk automatically instead of trying yo update the yum database each time it's called.